bundles/org.openhab.binding.asuswrt/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="tapocontrol" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+	<type>binding</type>
+	<name>Asuswrt Binding</name>
+	<description>Binding for ASUS routers (Asuswrt / Asuswrt-Merlin only)</description>
```

Comment:
If i'm right the binding does not rely on a cloud, so you can add `<connection>local</connection>`